packages/@repo/e2e/README.md

```diff
@@ -4,7 +4,10 @@
 
 The tests expect to find the below env variables. Either define them in your shell, or add them to the `.env.local` file in the repository root.
 
-- `SDK_E2E_SESSION_TOKEN`: As a developer running locally, you should use a user token. Running `sanity debug --secrets` will give you your token provided you are logged in (`sanity login`).
+- `SDK_E2E_SESSION_TOKEN`: As a developer running locally, you should use a user token. The client fixture also needs to use this token. Running `SANITY_INTERNAL_ENV=staging sanity debug --secrets` will give you your token provided you are logged in (`sanity login`).
```

Comment:
Dumb question: what's a client fixture?

Reply:
This will be a client that's handed to you as part of our custom test suite. you can see usage here: https://github.com/sanity-io/sdk/pull/560/files (clients.primary.patch)

Reply:
Ahh gotcha, thank you!